node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/DiskReplacer.java

```diff
@@ -39,10 +34,13 @@ public class DiskReplacer extends NodeRepositoryMaintainer {
 
     @Override
     protected double maintain() {
-        NodeList nodes = nodeRepository().nodes().list().rebuilding(true);
+        NodeList candidates = nodeRepository().nodes().list().rebuilding(true);
+        if (candidates.isEmpty()) {
+            return 0;
```

Comment:
Not 1?

Reply:
The definition has changed. 0 now indicates there is no work to do.